backend/controllers/index.ts

```
@@ -0,0 +1,26 @@
+import api from '../apis/index';
+
+const resetConnection = (res: { connection: { end: () => any } }) => {
+	res.connection && res.connection.end(); // Reset the connection pool
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)